### ISSUE TITLE
Add support to ControlNet

### ISSUE THREAD

**Issue by Warvito** [Project-MONAI/GenerativeModels#296]
In order to make possible the fine-tunning of pretrained diffusion models into specific tasks that contain images and labels maps as conditionings, it would be great to add the support to use hypernetworks on them, similar to the ControlNet (https://arxiv.org/abs/2302.05543)
